hive/src/main/java/io/confluent/connect/storage/hive/HiveSchemaConverter.java

```diff
@@ -26,11 +29,21 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import org.apache.kafka.connect.data.Time;
+import org.apache.kafka.connect.data.Timestamp;
 
 public class HiveSchemaConverter {
 
   private static final Map<Type, TypeInfo> TYPE_TO_TYPEINFO;
 
+  // the name here is consistent with io.confluent.connect.avro.AvroData, when connect Decimal
+  // schema is created, this property name is used to set precision.
+  // We have to use the exact name to retrieve precision value.
+  static final String CONNECT_AVRO_DECIMAL_PRECISION_PROP = "connect.decimal.precision";
```

Comment:
unfortunate that this is not public in `AvroData` but instead just package private. Let's not repeat the same mistake here.